src/power_divergence.jl

```diff
@@ -214,6 +235,48 @@ end
 
 # Under regularity conditions, their asymptotic distributions are all the same (Drost 1989)
 # Chi-squared null approximation works best for lambda near 2/3
+"""
+    PowerDivergenceTest(x [,y] [, lambda] [,theta0] )
+
+Perform a Power Divergence Test.
+
+If `x` is a matrix with one row or column, or if `x` is a vector and `y` is not given, then
```

Comment:
It's not clear whether "`y` is not given" applies to both cases or only to the first one.